packages/module/src/TagCount/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default } from './TagCount';
+export { default as TagCount } from './TagCount';
```

Comment:
```suggestion
export * from './TagCount';
```
let's not use named exports